index.html

```diff
@@ -221,6 +221,42 @@ <h3>Skills</h3>
                         </ul>
                     </article>
                 </article>
+                <article class="feedback">
+                    <p class="feedback-text">
+                        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                    </p>                    
+                    <form class="feedback-form" action="https://formspree.io/your-email" method="POST">
+                        <input type="text" id="fullname" name="fullname" placeholder="Full name" maxlength="30" required>
+                        <hr class="divider">
+                        <input type="email" id="email" name="email" placeholder="Email address" required>
+                        <hr class="divider">
+                        <div class="rectangle-border">
+                            <textarea id="message" name="message" rows="5" placeholder="Write me something" maxlength="500" required></textarea>
+                        </div>                        
+                        <button class="c-button" type="button">Get in touch</button>
+                    </form>
```

Comment:
Good implementation of your contact form. 👍🏿 However there are some changes I would suggest you make. ♻️ 

Currently your form does not submit because your form `action` does not have the correct `url`. Your `url` should be in a format below ⬇️ 
`https://formspree.io/hsjsjskl` where `hsjsjskl` is a unique set of alphabets generated by [Formspree](https://formspree.io/) after login and creating a form.

- [ ] Kindly login to [Formspree](https://formspree.io/) and create a form and use the link as the action for your form.

Reply:
Thank you for the feedback and the guidance provided.